packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -60,6 +24,15 @@ class Product extends Component {
         window.scrollTo(0, 0);
     }
 
+    // map Magento 2.3.1 schema changes to v2.0.0 proptype shape to maintain backwards compatibility
+    mapProduct(product) {
```

Comment:
Minor consistency issue here with how class functions are defined. Line 18 above uses a function expression instead.

Reply:
Actually I'm not sure why that function even exists. Doesn't seem like we're using addToCart anywhere, instead we're using the connected prop `addItemToCart`.

Reply:
But I think you're right, we should be using the class function rather than prototype otherwise `this.mapFoo` won't work.

Reply:
Oh wow, good catch @sirugh, that function is a good candidate for removal. I'm not keen on what's @api yet; can we safely remove it in a minor?

And who do you mean is right? The `mapProduct` function is fine as-is or Andy's suggestion that we should use expressions? My understanding was that function expressions were cleaner than binding params in constructor, but that's solved by the new shorthand.

Reply:
```js
class Foo {
  constructor() {
    	console.log(this);
  }
  first () {
    	console.log('first');
  }
  second = () => {
	console.log('second');
  }
}

const foo = new Foo() // "Foo { second: f }"
```

Reply:
The only reason to use arrow functions when defining class members is if that function will be passed directly as a value, usually to a React component prop. Let's definitely not be lazy about this--every arrow function definition is O(n), whereas every class member is O(1).